csbdeep/utils/tf.py

```diff
@@ -159,8 +159,8 @@ def _export(model):
                 _model.set_weights(weights)
                 _export(_model)
 
-        if meta is not None and len(meta) > 0:
-            save_json(meta, os.path.join(dirname,'meta.json'))
+    ##    if meta is not None and len(meta) > 0:
+    ##        save_json(meta, os.path.join(dirname,'meta.json'))
```

Comment:
```suggestion
        if meta is not None and len(meta) > 0:
            save_json(meta, os.path.join(dirname,'meta.json'))
```

Reply:
Let's not change this file.

Am I allowed to edit the code in your PR?

Reply:
Absolutely, please do so if necessary.